scripts/build-docker-images

```diff
@@ -56,13 +57,23 @@ for os_arch in "${PLATFORMS[@]}"; do
     dockerfile="$DOCKERFILE_PATH"
     if [[ $os = "windows" ]]; then
         dockerfile="${dockerfile}.windows"
+        docker build \
+            --file "${dockerfile}" \
+            --build-arg GOOS=${os} \
+            --build-arg GOARCH=${arch} \
+            --build-arg GOPROXY=${GOPROXY} \
+            --tag ${img_tag} \
+            ${REPO_ROOT_PATH}
+    else
+        # Launch a docker buildx instance and save its name so we can terminate it later
+        buildx_instance_name=$(docker buildx create --use)
+        docker buildx build \
```

Comment:
does `buildx` need to be setup on the gha runner or is it already installed?

Reply:
It was already installed. I didn't have to modify the gha runners and they are able to build successfully.